### PR TITLE
Blog app - Views

### PR DESCRIPTION
**Pull request by olawale-o**
## In this PR, I did the following:

- Style pages

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -1,5 +1,7 @@
 class UsersController < ApplicationController
   def index; end
 
-  def show; end
+  def show
+    @user_id = params[:id]
+  end
```
Comment (Emmyn5600) [inline app/controllers/users_controller.rb:6]: - [ ] Optional suggestion
This works perfectly, however, I would recommend this way `@user_id = User.find(params[:id])`

**Review by Emmyn5600 — APPROVED** [review]
Hi @olawale-o 

Good job so far! 👏🏼

## STATUS: APPROVED :muscle::clap:
- The project is running successfully
- The database works perfectly
- The views display the information.

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the project requirements.
user-1
user-2


 #### Optional suggestions
 
 - _Check the comment under the review_
 
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (olawale-o) [reply app/controllers/users_controller.rb:6]: Thank you for this helpful recommendation.

**Comment by olawale-o** [comment]
Hi @Emmyn5600 
Thank you for your time and effort on the review

**Pull request merged**